.env.test

```diff
@@ -0,0 +1,5 @@
+# define your env variables for the test env here
+KERNEL_CLASS='App\Kernel'
+APP_SECRET='$ecretf0rt3st'
+SYMFONY_DEPRECATIONS_HELPER=999999
+PANTHER_APP_ENV=panther
```

Comment:
What is PANTHER_APP_ENV=panther

Reply:
This file is generated by symfony framework. 
https://github.com/symfony/recipes/pull/643/files  
it's something to flex, but I don't really know what to do. 

I assumed it's better to leave it than for someone to bother about something for 3 days because tests doesn't work.

Reply:
Symfony Phanter is some kind of end2end test application